stable_baselines/a2c/a2c.py

```diff
@@ -192,10 +193,18 @@ def _train_step(self, obs, states, rewards, masks, actions, values, update, writ
 
         td_map = {self.train_model.obs_ph: obs, self.actions_ph: actions, self.advs_ph: advs,
                   self.rewards_ph: rewards, self.learning_rate_ph: cur_lr}
+        if len(action_masks) == 0:
+            action_masks = None
+        if action_masks is not None:
+            action_masks = reshape_action_mask(action_masks, self.train_model.ac_space, self.n_steps * self.n_envs)
         if states is not None:
             td_map[self.train_model.states_ph] = states
             td_map[self.train_model.dones_ph] = masks
-
+            if action_masks is None:
+                action_masks = create_dummy_action_mask(self.train_model.ac_space, states.shape[0] * self.n_steps)
```

Comment:
why not using [placeholder with default value](https://www.tensorflow.org/api_docs/python/tf/placeholder_with_default) here?